### PR TITLE
feat: add documents tab in org settings

### PR DESCRIPTION
**Pull request by pamelachia**
## What kind of change does this PR introduce?

in org settings, users can access a documents tab. For teams and enterprise plans they can access SOC2, security questionnaire, and HIPAA requests here. 

Waiting to be in prod: 
- https://github.com/supabase/infrastructure/pull/15245
- https://github.com/supabase/infra-perms/pull/118

Screenshot 2023-10-19 at 12 52 19 AM

### CODE REVIEW

File: studio/components/interfaces/Organization/Documents/DPA.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  ScaffoldSectionDetail,
+} from 'components/layouts/Scaffold'
+import { Button, IconExternalLink } from 'ui'
+
+const DPA = () => {
+  return (
+    <>
+      <ScaffoldSection>
+        <ScaffoldSectionDetail>
+          <div className="sticky space-y-6 top-12">
```
Comment (MildTomato) [inline studio/components/interfaces/Organization/Documents/DPA.tsx:13]: you can use classname on the components here, like `<ScaffoldSectionDetail>`

don't need to keep adding more divs inside

File: studio/components/interfaces/Organization/Documents/DPA.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+            <p className="text-base m-0">Data Processing Addendum (DPA)</p>
```
Comment (MildTomato) [inline studio/components/interfaces/Organization/Documents/DPA.tsx:14]: there's possibly a `<ScaffoldDetailTitle>` component

File: studio/components/interfaces/Organization/Documents/DPA.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
+import {
+  ScaffoldSection,
+  ScaffoldSectionContent,
+  ScaffoldSectionDetail,
+} from 'components/layouts/Scaffold'
+import { Button, IconExternalLink } from 'ui'
+
+const DPA = () => {
+  return (
+    <>
```
Comment (MildTomato) [inline studio/components/interfaces/Organization/Documents/DPA.tsx:10]: unrequired fragment

File: studio/components/interfaces/Organization/Documents/DPA.tsx (hunk 2 above)
Comment (pamelachia) [reply studio/components/interfaces/Organization/Documents/DPA.tsx]: Sorry couldn't find this. I tried` <ScaffoldTitle>` and `<ScaffoldHeader>` but it looks inconsistent with the other tabs. Maybe help suggest a change if you really need it as a different component?

File: studio/components/interfaces/Organization/Documents/DPA.tsx [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+import { Button, IconExternalLink } from 'ui'
+
+const DPA = () => {
+  return (
+    <ScaffoldSection>
+      <ScaffoldSectionDetail className="sticky space-y-6 top-12">
+        <p className="text-base m-0">Data Processing Addendum (DPA)</p>
+        <div className="space-y-2 text-sm text-foreground-light m-0">
+          <p>
+            For the document to be considered executed, sign and return the signed DPA document to{' '}
```
Comment (saltcod) [inline studio/components/interfaces/Organization/Documents/DPA.tsx:15]: Could reverse order here to get rid of the comma:

Sign and return the DPA document for it to be considered executed.

File: studio/components/interfaces/Organization/Documents/DPA.tsx [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+              href="mailto:privacy@supabase.com"
+              target="_blank"
+              className="text-brand hover:text-brand"
+            >
+              privacy@supabase.com
+            </a>
+            .
+          </p>
+          <p>
+            This is only to access the document, please ignore this if you have sent the signed
```
Comment (saltcod) [inline studio/components/interfaces/Organization/Documents/DPA.tsx:26]: This is only to access the document. Please ignore this if....

File: studio/components/interfaces/Organization/Documents/HIPAA.tsx [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+  const currentPlan = subscription?.plan
+
+  return (
+    <>
+      <ScaffoldSection>
+        <ScaffoldSectionDetail className="sticky space-y-6 top-12">
+          <p className="text-base m-0">HIPAA</p>
+          <div className="space-y-2 text-sm text-foreground-light m-0">
+            <p>
+              For organizations on Teams plan or above, you can request for a paid HIPAA add-on.
```
Comment (saltcod) [inline studio/components/interfaces/Organization/Documents/HIPAA.tsx:31]: Organizations on the Teams plan or above can request for a paid HIPPA add-on.

File: studio/components/interfaces/Organization/Documents/SOC2.tsx [hunk 7]
```diff
@@ -0,0 +1,133 @@
...
+    }
+  }
+
+  return (
+    <ScaffoldSection>
+      <ScaffoldSectionDetail className="sticky space-y-6 top-12">
+        <p className="text-base m-0">SOC2 Type 2</p>
+        <div className="space-y-2 text-sm text-foreground-light m-0">
+          <p>
+            For organizations on Teams plan or above, you have access to our most recent SOC2 Type 2
```
Comment (saltcod) [inline studio/components/interfaces/Organization/Documents/SOC2.tsx:49]: Organizations on the Teams plan or above have access to our most recent SOC2 Type 2 report.  (?)

File: studio/components/interfaces/Organization/Documents/SecurityQuestionnaire.tsx [hunk 8]
```diff
@@ -0,0 +1,90 @@
...
+  }
+
+  return (
+    <>
+      <ScaffoldSection>
+        <ScaffoldSectionDetail className="sticky space-y-6 top-12">
+          <p className="text-base m-0">Standard Security Questionnaire</p>
+          <div className="space-y-2 text-sm text-foreground-light m-0">
+            <p>
+              For organizations on Teams plan or above, you have access to our standard security
```
Comment (saltcod) [inline studio/components/interfaces/Organization/Documents/SecurityQuestionnaire.tsx:49]: Organizations on the Teams plan or above have access to our .....

**Review by saltcod — APPROVED** [review]

**Pull request merged**